internal/config/db.go

```diff
@@ -59,6 +59,9 @@ type Postgres struct {
 type PostgresTLS struct {
 	CABundlePath string `yaml:"ca_bundle_path" json:"ca_bundle_path" env:"CA_BUNDLE_PATH"`
 	Enable       bool   `yaml:"enable" json:"enable" env:"ENABLE"`
+	// SslMode should be one of the postgresql ssl modes. Default to 'verify-full'.
+	// Ref: https://www.postgresql.org/docs/current/libpq-ssl.html#LIBPQ-SSL-SSLMODE-STATEMENTS
+	SslMode string `yaml:"ssl_mode" json:"ssl_mode" env:"SSL_MODE"`
```

Comment:
If we'd want to keep this, these fields should be named `SSLMode`:
https://github.com/golang/go/wiki/CodeReviewComments#initialisms